net/matrix-structs/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+
+github.setup        mujx matrix-structs 8de04af
+version             2018-07-14
+categories          net devel
+platforms           darwin
+license             public-domain
+maintainers         {@scarface-one} openmaintainer
```

Comment:
A maintainer line needs to contain both an email address (in the obscured form you will see in other portfiles) and the github handle. (If you're reluctant for some reason to put your obscured email in, we could go back to leaving this "nomaintainer".)

Reply:
fixed